ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -117,7 +118,7 @@ class MSFListCellTokens: MSFTokensBase, ObservableObject {
         disclosureIconForegroundColor = appearanceProxy.disclosureIconForegroundColor
         leadingTextColor = appearanceProxy.labelColor
         leadingViewColor = appearanceProxy.leadingViewColor
-        subtitleColor = appearanceProxy.sublabelColor
+        sublabelColor = appearanceProxy.sublabelColor
```

Comment:
Since you're changing names of the properties, can you normalize all properties as "label"? (I see subtitleFont in line 138)

Reply:
Same for textFont vs labelFont in line 139.

Reply:
I've updated the property names but left title/subtitle/footnote in the cell state. I figured we should distinguish that sub labels include subtitle + footnote but that they are still separate properties

Reply:
Okay. Something else I noticed as you renamed leadingTextColor to leadinglabelColor:
Why is it not just labelColor?

Reply:
Oh good catch. It was a remnant from when we used to have a trailing label value. Updated, thank you